### ISSUE TITLE
Yum upgrade of 4.0 to 4.1-preview fails

### ISSUE THREAD

**Issue by utopiabound** [whamcloud/integrated-manager-for-lustre#707]
### Description 

python-iml-manager and python-iml-manager-libs do not obsolete their chroma-manager counterparts.

### Repro

Setup:
```
yum -y install yum-plugins-copr
yum copr enable managerforlustre/manager-for-lustre-devel
```
Attempted Upgrade:
```
# yum install python2-iml-manager.noarch python2-iml-manager-libs.noarch
Loaded plugins: copr, fastestmirror, langpacks
Repository extras is listed more than once in the configuration
Loading mirror speeds from cached hostfile
 * base: mirror.nodesdirect.com
 * epel: fedora-epel.mirror.iweb.com
 * extras: mirror.wdc1.us.leaseweb.net
 * updates: mirror.dal10.us.leaseweb.net
Resolving Dependencies
--> Running transaction check
---> Package python2-iml-manager.noarch 0:4.1.0.0-5114.13.g9da817a.el7 will be installed
--> Processing Dependency: python2-iml-manager-cli = 4.1.0.0-5114.13.g9da817a.el7 for package: python2-iml-manager-4.1.0.0-5114.13.g9da817a.el7.noarch
--> Processing Dependency: iml-update-handler < 2 for package: python2-iml-manager-4.1.0.0-5114.13.g9da817a.el7.noarch
--> Processing Dependency: iml-device-scanner-aggregator for package: python2-iml-manager-4.1.0.0-5114.13.g9da817a.el7.noarch
--> Processing Dependency: iml-old-gui for package: python2-iml-manager-4.1.0.0-5114.13.g9da817a.el7.noarch
--> Processing Dependency: iml-realtime for package: python2-iml-manager-4.1.0.0-5114.13.g9da817a.el7.noarch
--> Processing Dependency: iml-socket-worker for package: python2-iml-manager-4.1.0.0-5114.13.g9da817a.el7.noarch
--> Processing Dependency: iml-view-server for package: python2-iml-manager-4.1.0.0-5114.13.g9da817a.el7.noarch
---> Package python2-iml-manager-libs.noarch 0:4.1.0.0-5114.13.g9da817a.el7 will be installed
--> Processing Dependency: python2-iml-common1.4 for package: python2-iml-manager-libs-4.1.0.0-5114.13.g9da817a.el7.noarch
--> Running transaction check
---> Package iml-device-scanner-aggregator.x86_64 0:2.2.0-197.el7 will be installed
---> Package iml-old-gui.noarch 0:3.0.0-39.el7 will be installed
---> Package iml-realtime.x86_64 0:7.0.0-67.el7 will be installed
---> Package iml-socket-worker.noarch 0:4.0.0-98.el7 will be installed
---> Package iml-update-handler.noarch 0:1.0.0-24.12.gb359059.el7 will be installed
---> Package iml-view-server.noarch 0:8.0.1-141.el7 will be installed
---> Package python2-iml-common1.3.noarch 0:1.3.3-1.el7.centos will be obsoleted
--> Processing Dependency: python2-iml-common1.3 for package: chroma-manager-libs-4.0.6.0-5034.el7.x86_64
---> Package python2-iml-common1.4.noarch 0:1.4.4-43.2.el7 will be obsoleting
---> Package python2-iml-manager-cli.noarch 0:4.1.0.0-5114.13.g9da817a.el7 will be installed
--> Finished Dependency Resolution
Error: Package: chroma-manager-libs-4.0.6.0-5034.el7.x86_64 (@chroma-manager)
           Requires: python2-iml-common1.3
           Removing: python2-iml-common1.3-1.3.3-1.el7.centos.noarch (@managerforlustre-manager-for-lustre)
               python2-iml-common1.3 = 1.3.3-1.el7.centos
           Obsoleted By: python2-iml-common1.4-1.4.4-43.2.el7.noarch (managerforlustre-manager-for-lustre-devel)
              ~python-iml-common1.4 = 1.4.4-43.2.el7
              ~python2-iml-common1.4 = 1.4.4-43.2.el7
 You could try using --skip-broken to work around the problem
 You could try running: rpm -Va --nofiles --nodigest
```
### Version
el7.5 (CentOS 7.5)
IML 4.0.6.0

**Comment by utopiabound** [comment]
This will probably be fix by same patch that fixes #682

**Comment by jgrund** [comment]
There is a start at fixing both of those in: https://github.com/whamcloud/integrated-manager-for-lustre/pull/681
